TRAMS-API/Mapping/Request/PostProjectAcademiesRequestMapper.cs

```diff
@@ -0,0 +1,32 @@
+﻿using API.Models.Downstream.D365;
+using API.Models.Upstream.Request;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace API.Mapping.Request
+{
+    public class PostProjectAcademiesRequestMapper : IMapper<PostProjectsAcademiesModel, PostAcademyTransfersProjectAcademyD365Model>
+    {
+        public PostAcademyTransfersProjectAcademyD365Model Map(PostProjectsAcademiesModel input)
+        {
+            return new PostAcademyTransfersProjectAcademyD365Model
+            {
+                AcademyId = $"/accounts({input.AcademyId})",
+                EsfaInterventionReasons = input.EsfaInterventionReasons != null && input.EsfaInterventionReasons.Any()
```

Comment:
Suggestion: I've seen this pattern used quite a lot and was thinking maybe you could create an extension method for List<string> (or whatever the collection is) so that you can just do input.EsfaInterventionReasons.ToDelimitedString() and input.RddOrRscInterventionReasons.ToDelimitedString() for example.

Reply:
Good idea, will surely help with the readability of these things

Reply:
done